service/synapse-service-rest/src/main/java/io/americanexpress/synapse/service/rest/controller/reactive/BaseReactiveCreateController.java

```diff
@@ -57,11 +52,13 @@
             @ApiResponse(code = 401, message = "Unauthorized"),
             @ApiResponse(code = 403, message = "Forbidden"),
     })
-    public ResponseEntity<Mono<O>> create(@Valid @RequestBody I serviceRequest) {
+    public Mono<ResponseEntity<O>> create(@Valid @RequestBody I serviceRequest) {
         logger.entry(serviceRequest);
 
-        final O serviceResponse = service.create(serviceRequest);
-        ResponseEntity<Mono<O>> responseEntity = reactiveCreateResponseEntityCreator.create(serviceResponse);
+        final var serviceResponse = service.create(serviceRequest);
+        final var responseEntity = serviceResponse
+                .map(ResponseEntity::ok)
+                .defaultIfEmpty(ResponseEntity.noContent().build());
```

Comment:
Should it be this because for create it should return HttpStatus.CREATED instead of OK?
```suggestion
        final var responseEntity = serviceResponse.map(ResponseEntity.status(HttpStatus.CREATED)::body)
                        .defaultIfEmpty(ResponseEntity.internalServerError().build());
```